composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require": {
         "php": ">=7.0",
-        "nikic/php-parser": "^3.0.4",
+        "nikic/php-parser": "^3.0.5",
```

Comment:
@staabm that is not correct. There is no composer.lock committed (best practice for libraries).